module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -1729,6 +1758,23 @@ public function recoverAction()
         } elseif ($username = $this->params()->fromPost('username')) {
             $user = $userService->getUserByUsername($username);
         }
+        //ILS Driver:
+        //if the user hasn't logged in yet, but is found by the ILS, call function
+        //getPatronFromBarcode
```

Comment:
getPatronFromBarcode isn't a standard ILS driver method. Is this something you are planning to add, or is this local code that doesn't apply upstream? If this is something you plan to introduce officially, I'd discourage use of the word "Barcode" since different systems will use different types of identifiers...

Reply:
renamed barcode to username